### PR TITLE
fix #7 - take into account line returns inside input

### LINKED ISSUE #7: Take into account line returns inside input
I think this module should take into account line returns already present into the input.

For example, currently:

```js
var input = '12345678\n901234567890';

console.log(wrapAnsi(input, 10, {hard: true}));
```

will print

```
12345678
9
0123456789
0
```

I'd expect:

```
12345678
9012345678
90
```

### PR DESCRIPTION
**Pull request by dthree**

### CODE REVIEW

**Comment by dthree** [comment]
Peer review please before I merge!

File: test.js [hunk 1]
```diff
@@ -87,3 +88,12 @@ test('does not prepend newline if first word is split', function (t) {
 	t.assert(res.split('\n').length === 2);
 	t.end();
 });
+
+test('takes into account line returns inside input', function (t) {
+	var res20 = fn(fixture2, 10, {hard: true});
+	t.assert(res20 === '12345678\n9012345678\n90');
+	t.assert(stripAnsi(res20).split('\n').every(function (x) {
```
Comment (SBoudrias) [inline test.js]: That assertion is probably not very useful as you're asserting the full content of the string first.

**Comment by SBoudrias** [comment]
LGTM

File: test.js (hunk 1 above)
Comment (dthree) [inline test.js]: Hmm. Not sure what you mean by that? The second (comparative) string has an added line break near the end that only this method would properly handle, I believe.

**Comment by dthree** [comment]
KK. @bcoe I can merge this, but you'll have to publish - don't think I have admin rights to publish the npm module.

**Comment by sindresorhus** [comment]
@dthree You do now. Use [np](https://github.com/sindresorhus/np) for publishing.

**Comment by dthree** [comment]
Thanks! :+1:

**Comment by bcoe** [comment]
@dthree \o/ can't wait to see if this lets me eliminate some of the weird logic in:

https://github.com/bcoe/yarsay

I'll test this against `cliui` and `yargs` later tonight -- perhaps we could make this a major version bump, so that folks don't get this new behavior until we've thoroughly tested with the dependent libraries, what are your thoughts (I could go either way).

**Comment by dthree** [comment]
It's funny that a bug fix becomes a major bump, but I have to agree with you on that one @bcoe 

2.0 it is.

**Pull request merged**